src/pages/index.js

```diff
@@ -29,14 +33,21 @@ const IndexPage = () => {
         What is a City Challenge?
         <br />
         <h2 style={rightBlurbs}>
-        A City Challenge is a public engagement model in which city leaders and officials work with residents to identify, design and implement resident-designed solutions to some of the city’s most pressing problems.                                                    
+          A City Challenge is a public engagement model in which city leaders and officials work with residents to identify, design and implement resident-designed solutions to some of the city’s most pressing problems.                                                    
         <h3 style={rightBlurbs}></h3>
-        The goal of a City Challenge is to help a city transition from closed-door to open and collaborative governance, and helps governments and communities co-create implementable solutions to critical local problems in twelve months through a 4-step process. 
+          The goal of a City Challenge is to help a city transition from closed-door to open and 
+          <ResponsiveDialog props={{word: 'collaborative governance', def: 'A process involving state and non-state actors jointly addressing an issue.'}} textStyles={wordDef}></ResponsiveDialog>, 

```

Comment:
If you change `textStyles` to be included in `props`, then you can just do:

```suggestion
          <ResponsiveDialog word={'collaborative governance'} def={'A process involving state and non-state actors jointly addressing an issue.'} textStyles={wordDef} />, 
```